tracks/nit/nitcc/config.ini

```diff
@@ -1,2 +1,3 @@
 title=Elvish Vectorial Images and nitcc
 req=visitor
+languages=plain
```

Comment:
> Maybe just refuse the load tracks that does not set the languages?

I think this is the best behavior to adopt for now. Can you make the change in this PR?